src/System.Transactions/tests/System.Transactions.Tests.csproj

```diff
@@ -15,6 +15,10 @@
     <Compile Include="TransactionScopeTest.cs" />
     <Compile Include="AsyncTest.cs" />
     <Compile Include="EnlistTest.cs" />
+    <Compile Include="CloneTxTests.cs" />
+    <Compile Include="LTMEnlistmentTests.cs" />
+    <Compile Include="HelperFunctions.cs" />
+    <Compile Include="TestEnlistments.cs" />
```

Comment:
I think I now see how the system.transactions became lowercase. Do I need to delete and re-add them with the correct casing?

---
In reply to: [72360365](https://github.com/dotnet/corefx/pull/10350#discussion_r72360365) [](ancestors = 72360365,72360242,72359008)